<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortDebugger</class>
 <widget class="QMainWindow" name="SerialPortDebugger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SerialPortDebugger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" rowspan="2" colspan="3">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="0" column="3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>串口配置</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="Label">
         <property name="text">
          <string>串口号</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_SerialPortName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Label_2">
         <property name="text">
          <string>波特率</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_BaudRate">
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="Label_3">
         <property name="text">
          <string>数据位</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_CharacterSize">
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="Label_4">
         <property name="text">
          <string>校验位</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBox_Parity">
         <item>
          <property name="text">
           <string>none</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>odd</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>even</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="Label_5">
         <property name="text">
          <string>停止位</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboBox_StopBits">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="Label_6">
         <property name="text">
          <string>流控制</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboBox_FlowControl">
         <item>
          <property name="text">
           <string>none</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>software</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>hardware</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_SerialPortControl">
         <property name="text">
          <string>打开串口</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="checkBox_3">
      <property name="text">
       <string>自动发送</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="spinBox">
      <property name="suffix">
       <string>ms</string>
      </property>
      <property name="maximum">
       <number>600000</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineEdit_UserMessage"/>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QPushButton" name="pushButton_SendMessage">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" rowspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>功能区</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>37</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Hex发送</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>Hex接收</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>43</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
